docs/metrics.md

```diff
@@ -164,6 +168,45 @@ Open a web browser to [http://127.0.0.1:3000](http://127.0.0.1:3000), you should
 
 > Makefile targets `make grafana-portforward`,`make kind-grafana-portforward` and `make minikube-grafana-portforward`.
 
+### Stackdriver installation
```

Comment:
This will need to get moved over to the new website docs :cry:  sorry - someone was going to get bit by this.